examples/official-storybook/components/FlowTypeButton.js

```diff
@@ -2,11 +2,11 @@
 import React from 'react';
 
 type PropsType = {
-  /** The text to be rendered in the button */
-  label: string,
+  /** A multi-type prop to be rendered in the button */
+  label: string | number,
```

Comment:
Can you please add here as much different types as possible? At the very least, we should repeat all the types present in DocgenButton, to ensure that they work with flow. But there's probably a sense in trying to add some more complex types as well (because when you use flow, usually there are more complex types around)